Project-II-Calculator/calculator/src/components/ButtonComponents/NumberButton.js

```diff
@@ -1,2 +1,20 @@
 import React from 'react';
 import './Button.css';
+
+function NumberButton(props) {
+  const { button } = props;
+
+  let buttonStyleWidth = button.value === 'clear' || button.value === '0'
```

Comment:
This is really cool. It shows you understand how to use props and make your components reusable. I think you would enjoy styled components.